Exercises/variables-declaration/variables-declaration.js

```diff
@@ -0,0 +1,43 @@
+// 1. Declare a variable which will be encapsulated from global scope.
+// 3. Declare two variables which will be same-names but contain different values.
+let firstOne = 0;
+let secondOne = 'Some number';
+console.log(secondOne);
```

Comment:
1. Are these vars really encapsulated from global (the top-most) scope? (It's not the same that to be not present in global object).
2. Which two variables have the same names but contain different values here?

Reply:
Committed some changes, hope now it's better)

Reply:
Thought that re-asigning the variable in the loop is the same

Reply:
Oh, I understand my mistake about encapsulation.
I encapsulated the value of variable in the loop, but not the variable itself